WinImageBuilder.psm1

```diff
@@ -537,6 +547,22 @@ function Add-VirtIODrivers {
 }
 
 function Add-VirtIODriversFromISO {
+    <#
+    .SYNOPSIS
+     The VirtIO ISO contains all the synthetic drivers for the KVM hypervisor. 
+     This function allows you to add VirtIO drivers to your Windows image, from an ISO file.
+    .DESCRIPTION
+     This function takes the VirtIO drivers from a specified ISO file and installs them into the 
+     given VHD, based on the characteristics given by the image parameter (which contains the
+     image version, image architecture and installation type).
+    .PARAMETER VHDDriveLetter
+     The drive letter of the mounted Windows image.
+    .PARAMETER Image
+     The exact flavor of Windows installed on that image, so that the appropriate VirtIO drivers
+     can be installed.
+    .PARAMETER ISOPath
+     The path to the VirtIO ISO file, where the drivers can be found.

```

Comment:
I would change it to:

 `The full path of the VirtIO ISO file containing the drivers.`